tests/python/pants_test/backend/python/tasks/BUILD

```diff
@@ -17,6 +17,28 @@ python_library(
   ]
 )
 
+python_tests(
+  name='python_native_code_testing',
+  sources=[
```

Comment:
These should be excluded from the `tasks` target; we don't want to run them twice on linux

Reply:
Done by adding the python variable `python_native_code_test_files` to use as sources in the platform-specific test target, and excludes in the generic unit and integration testing targets in that directory. Let me know if there is a better BUILD file pattern I could be using.